src/driver/odoc_units_of.ml

```diff
@@ -264,11 +276,23 @@ let packages ~dirs ~extra_paths ~remap ~gen_indices (pkgs : Packages.t list) :
         [ Landing_pages.src ~dirs ~pkg ~index ]
       else []
     in
-    List.concat
-      ((pkg_index :: src_index :: lib_units)
-      @ mld_units @ asset_units @ md_units)
+    match indices_style with
+    | Normal | Voodoo ->
+      List.concat
+        ((pkg_index () :: src_index () :: lib_units)
+        @ mld_units @ asset_units @ md_units)
+    | Custom ->
+      if pkg.name = "pkg" then
+        let others :> t list = Landing_pages.make_custom dirs index_of (List.find (fun p -> p.Packages.name = "pkg") pkgs) in
+        others @ List.concat
+          (mld_units @ asset_units @ md_units @ lib_units)
+      else
+        List.concat
+          ((pkg_index () :: src_index () :: lib_units)
+          @ mld_units @ asset_units @ md_units)
```

Comment:
When `pkg.name = "pkg"` is false, this returns the same thing as in the Normal or Voodoo case. The code could be made clearer.